lib/logMessages.js

```diff
@@ -0,0 +1,55 @@
+const TEMPLATE_INSTALL_FLAG_MSG = 'because you passed --install flag';
+
+const TEMPLATE_INSTALL_DISK_MSG = 'because the template cannot be found on disk';
+
+const NODE_MODULES_INSTALL ='Remember that your local template must have its own node_modules installed first, \"npm install\" is not triggered by the generator.';
+
+const NPM_INSTALL_TRIGGER = 'Installation of template located on disk technically means symlink creation betweed node_modules of the generator and template sources. Your local template must have its own node_modules, "npm install" is not triggered.';
+
+function templateVersion(ver) {  
+  return `Version of used template is ${ver}.`;
+} 
+
+function templateSource(localHtmlTemplate) {
+  return `Template sources taken from ${localHtmlTemplate}.`;
+} 
+
+function templateNotFound(templateName) {
+  return `${templateName} not found in local dependencies but found it installed as a global package.`;
+} 
+
+function packageNotAvailable(pkgPath) {
+  return `Unable to resolve template location at ${pkgPath}. Package is not available locally.`;
+}
+
+function installationDebugMessage(debugMessage) {
+  return `Template installation started ${debugMessage}`;
```

Comment:
please add `.` at the end of the sentence, same in line 1 and 3. Let us be consistent and have dot in all the log messages. With this simple change, you will see why it was worth the effort to do the refactor. Next time we do any improvements in log messages texts, we just do them here in one file, no need to touch main library code and tests.

Reply:
Yes ,you are absolutely right and this one change is significant as it maintains the consistency .There is so much I have learnt while working on thi issue and consistency in code is definitely up there.

Thank you.